pkgs/top-level/all-packages.nix

```diff
@@ -9593,6 +9593,8 @@ with pkgs;
 
   just = callPackage ../development/tools/just { };
 
+  justbuild = callPackage ../development/tools/build-managers/justbuild { };
```

Comment:
Please move the package to `pkgs/by-name/ju/justbuild/package.nix`, then remove this line.

Reply:
Done, thanks